model/src/main/kotlin/OrtResult.kt

```diff
@@ -19,12 +19,15 @@
 
 package com.here.ort.model
 
+import ch.frankel.slf4k.info
```

Comment:
We have the unwritten convention to use the wildcard import for this `import ch.frankel.slf4k.*`.

Reply:
done.